### PR TITLE
Chat

### PR DESCRIPTION
**Pull request by jody29**
Only 2 specific users should now be able to chat with eachother

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const dotenvExpand = require('dotenv-expand')
 const myEnv = dotenv.config()
 dotenvExpand(myEnv)
 
-const methodOverride = require('method-override');
```
Comment (Sophievanderburg) [inline index.js]: Oh oops, I am still adjusting haha.

File: public/scripts/convertMusic.js [hunk 2]
```diff
@@ -12,63 +12,65 @@ const spotify = new Spotify({
...
-			// console.log(artistArray);
+const convertMusic = (inputQuery) => {
+    return new Promise((resolve, reject) => {
+        spotify.search(
+            { type: 'track', query: inputQuery },
+            function (err, data) {
+                if (err) {
+                    return console.log(`Error occurred: ${err}`)
+                }
+                let songdata = data.tracks.items[0]
```
Comment (Sophievanderburg) [inline public/scripts/convertMusic.js]: I see @Djescho is also still adjusting to the new coding style :)

File: views/pages/saved_matches.ejs [hunk 3]
```diff
@@ -28,7 +28,7 @@
           <input value="<%= data[i]._id%>" type="hidden" name="userId" />
           <button type="submit" class="remove">Remove</button>
         </form>
-        <a href="/chat:id">Chat!</a>
+        <a href="/chat/1">Chat!</a>
```
Comment (Sophievanderburg) [inline views/pages/saved_matches.ejs:27]: Why do you use a number here instead of :id?

**Review by Sophievanderburg — CHANGES_REQUESTED** [review]
Look at the comment in the saved_matches.ejs file. Can you explain it to me?

File: views/pages/saved_matches.ejs (hunk 3 above)
Comment (jody29) [reply views/pages/saved_matches.ejs:27]: This was just for a test. Should be `/chat/<%= chatNumber %>` when it's totally working

**Pull request merged**